posts/introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+title: An introduction to X-GOVUK
+date: 2022-03-14
+tags: post
+layout: post
+authors:
+  - Frankie Roberto
+  - Paul Lloyd
+  - Paul Hayes
+---
+
+Hello! X-GOVUK is an unofficial community-maintained space for people working on government digital services in the UK.
+
+The "X" stands for "cross", as in "cross-government".
+
+Collaboration across department and organisational boundaries within government is often very tricky, for all sorts of practical and cultural reasons.
+
+But it can also be incredibly rewarding and save a lot of time, by sharing designs, code, and research findings.
+
+The "X" also stands for "experimental", as this can be a space for ideas that aren't yet proven.
+
+The [X-GOVUK](/) homepage contains a list of useful design, content, prototyping and front-end resources.
+
+Projects include:
+
```

Comment:
I think we need to distinguish between the list of projects from across government, and those maintained by X-GOVUK.

Reply:
Good spot, this no longer made sense since the list was added.

I've moved the sentence about the list on the homepage down - let me know what you think.